src/badge/badge.js

```diff
@@ -10,11 +10,28 @@ const Badge = props => {
 
   if (badge.element) return badge.element;
```

Comment:
instead of doing this, lets use prop de-structuring similar to those in other components for examples

```js
const {
  component,
  containerStyle,
  textStyle,
  title,
  ...attributes
} = props;

return (
  <Component style={[styles.badge, containerStyle && containerStyle]} {...attributes}>
      <Text style={[styles.text, textStyle && textStyle]}>{title}</Text>
  </Component>
)
```